### PR TITLE
Actually adding tests...but i just continued work on the branch

### PR DESCRIPTION
**Pull request by crmorford**

### CODE REVIEW

**Comment by bluong** [comment]
jshint is complaining.  Please run
`grunt jshint` before pushing.

**Comment by bluong** [comment]
While we do like tests, managing data files can be quite the hassle...  What do you think about what we should do with these data files? @jtlan

**Comment by crmorford** [comment]
oops, i missed your comment for this push @bluong , but will do next time

File: quicktests/overlaying/tests/realistic/animals.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
+
+
```
Comment (bluong) [inline quicktests/overlaying/tests/realistic/animals.js]: Remove the above lines

**Comment by bluong** [comment]
As an aside, we should probably up our jshint configurations so that our javascript files are decently neat.  I can see a random user stumbling upon these quicktests and we wouldn't want to present to them a decent mess.

**Comment by jtlan** [comment]
@bluong wrote:
>While we do like tests, managing data files can be quite the hassle...  What do you think about what we should do with these data files? @jtlan 

We can just carry around the .csv files for now. Later on, we can try to work out a better way to do it.

**Comment by crmorford** [comment]
closing for now, focusing on PRs

**Pull request closed without merging**